controller/bin/daq_config.yaml

```diff
@@ -1,6 +1,6 @@
 # DAQ FOT configuration.
 
-include: ../daq/config/system/default.yaml
+include: daq_system_default.yaml
```

Comment:
why create a new separate file?

Reply:
Reverted. I had some path issue with some DAQ version but now it is working. Not sure what fixed it...